plugin/Region/RegionController.php

```diff
@@ -100,6 +100,9 @@ private function createRegionObjectsRegionOne(array $regionList): array
 	{
 		$regions = [];
 		foreach ($regionList as $regionProperties) {
+			if (empty($regionProperties['elements']['id'])) {
```

Comment:
`!isset()` would be preferred over `empty()`. That's a minor thing, though.

Reply:
`empty()` includes checking that the variable exists and is not null. If the api returns id is null there will still be an error